src/NexusMods.App/Startup.cs

```diff
@@ -102,7 +102,7 @@ public static AppBuilder BuildAvaloniaApp(IServiceProvider serviceProvider)
             .LogToTrace()
             .UseR3()
             .UseReactiveUI()
-            .With(new SkiaOptions { UseOpacitySaveLayer = true });
+            .With(new SkiaOptions { UseOpacitySaveLayer = false });
```

Comment:
What happened here? As mentioned in the comment below by @insomnious, we need this setting to disabled/fade SVG icons.